plugins/adr-common/src/index.ts

```diff
@@ -57,11 +57,19 @@ export const isAdrAvailable = (entity: Entity) =>
 export const getAdrLocationUrl = (
   entity: Entity,
   scmIntegration: ScmIntegrationRegistry,
+  adrFilePath?: String,
 ) => {
   if (!isAdrAvailable(entity)) {
     throw new Error(`Missing ADR annotation: ${ANNOTATION_ADR_LOCATION}`);
   }
 
+  if (adrFilePath) {
+    return scmIntegration.resolveUrl({
+      url: `${getAdrLocationDir(entity)!.replace(/\/$/, '')}/${adrFilePath}`,
+      base: getEntitySourceLocation(entity).target,
+    });
+  }
+
   return scmIntegration.resolveUrl({
     url: getAdrLocationDir(entity)!,
     base: getEntitySourceLocation(entity).target,
```

Comment:
Can probably DRY this up a bit: 

```suggestion
  let url = getAdrLocationDir(entity)!;
  if (adrFilePath) {
    url = `${url.replace(/\/$/, '')}/${adrFilePath}`;
  }

  return scmIntegration.resolveUrl({
    url,
    base: getEntitySourceLocation(entity).target,
```